selenium/base/base_2/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN apt-get update && apt-get install -y ttf-mscorefonts-installer \
     xfonts-encodings \
     xfonts-utils \
     flashplugin-installer \
+    fluxbox \
```

Comment:
Adding fluxbox here means I have to recreate all browser images as they are all using this layer.

Reply:
The other alternative is to install fluxbox each by one in the browser images, but I think if you want to be used in all containers, it is a better to keep this package in the base.